libraries/commerce/product/selectors/product.js

```diff
@@ -453,25 +465,64 @@ export const getProductMetadata = createSelector(
 );
 
 /**
- * Indicates whether a product is a base product or not
  * @param {Object} state The current application state.
- * @param {string} productId A product id.
+ * @param {object} props The component props.
  * @return {boolean|null}
  */
 export const isBaseProduct = createSelector(
-  getProductById,
-  (product) => {
-    if (!product.productData || product.isFetching) {
+  getProducts,
+  (state, props) => props.id,
```

Comment:
Why is it suddenly `id` here? This is confusing. Please change it to be `productId` as well here.

Reply:
Indeed, is now productId